### PR TITLE
Bugfix/OP-1047: IE - Admin must hold CTL button to add permissions

### PR DESCRIPTION
**Pull request by gzhou1**
- Added messaging telling user to hold control to select multiple permissions if user is on Internet Explorer.

### CODE REVIEW

File: app/templates/response/add_user_request.html [hunk 1]
```diff
@@ -19,7 +19,10 @@
                 </div>
                 <div id="add_options" hidden>
                     {{ add_user_request_form.roles.label }}<br>
-                    {{ add_user_request_form.roles(id="role", class="form-control") }}
+                    {{ add_user_request_form.roles(id="role", class="form-control") }}<br>
+                    <div id="ie-message" class="alert alert-info" hidden><strong>NOTE:</strong> Hold control and click
```
Comment (Palisand) [inline app/templates/response/add_user_request.html]: control -> Ctrl, include Shift as well

Comment (gzhou1) [inline app/templates/response/add_user_request.html]: Updated.

**Comment by Palisand** [comment]
Can you please fix the difference in spacing for Edit User (between Users and Presets):

Good
screen shot 2017-03-29 at 2 20 52 pm

Bad
screen shot 2017-03-29 at 2 21 13 pm

File: app/templates/response/add_user_request.html [hunk 2]
```diff
@@ -19,7 +19,11 @@
                 </div>
                 <div id="add_options" hidden>
                     {{ add_user_request_form.roles.label }}<br>
-                    {{ add_user_request_form.roles(id="role", class="form-control") }}
+                    {{ add_user_request_form.roles(id="role", class="form-control") }}<br>
+                    <div id="ie-message" class="alert alert-info" hidden><strong>NOTE:</strong> Hold Ctrl and click
+                        to select multiple permissions one at a time. Hold Shift and click to select a range of
```
Comment (Palisand) [inline app/templates/response/add_user_request.html]: make Ctrl and Shift strong as well

Comment (gzhou1) [inline app/templates/response/add_user_request.html]: The spacing issue has been resolved and Ctrl and Shift have been made strong.

**Pull request merged**